packages/amplify-cli/src/__tests__/extensions/amplify-helpers/get-userpoolgroup-list.test.ts

```diff
@@ -0,0 +1,43 @@
+import { getUserPoolGroupList } from '../../../../src/extensions/amplify-helpers/get-userpoolgroup-list';
```

Comment:
I think you can simplify this a bit:

```suggestion
import { getUserPoolGroupList } from '../../../extensions/amplify-helpers/get-userpoolgroup-list';
```

Reply:
Thanks. I fixed it.